### ISSUE TITLE
Is DITA4 publishers picky about spaces in file folders? 

### ISSUE THREAD

**Issue by tommyr1962** [dita4publishers/dita4publishers#21]
Using DITA-OT1.8.5, the input path to my map had spaces in folder names. When I ran the "java -jar lib/dost.jar" command to generate ePub, all I got was the Help menu.When I replaced spaces with underscores, it worked.

**Comment by drmacro** [comment]
Yes, this is a known issue. Radu Coravu reported it and it is (should be) fixed in GitHub and will be in the next release. Basically I forgot to quote  path somewhere.

Note that you should really be using ant rather than lib/dost.jar to run the OT: the dost.jar is just a wrapper around Ant anyway, not that it should make a difference in this case.

**Comment by tommyr1962** [comment]
Thanks, Eliot .. I normally use Ant but I don't do much generating from the command line anyway.

Regarding the issue, I believe we encourage no spaces in our CMS, or we can until the next plug-in version. So it shouldn't be a problem at all. 

Thanks again, 

Tom
